release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -482,6 +482,13 @@ With this release, you can install a cluster on Nutanix by using the named, prel
 
 For more information, see xref:../installing/installing_nutanix/installation-config-parameters-nutanix.adoc#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix[Additional Nutanix configuration parameters].
 
+[id="ocp-4-18-installation-and-update-nutanix-multiple-nics_{context}"]
+==== Installing a cluster on Nutanix with up to 32 subnets
+With this release, Nutanix supports more than one subnet for the Prism Element to where the {product-title} cluster is deployed. A maximum of 32 subnets for each Prism Element is supported.
```

Comment:
```suggestion
With this release, Nutanix supports more than one subnet for the Prism Element to where an {product-title} cluster is deployed. A maximum of 32 subnets for each Prism Element is supported.
```

Reply:
Also reading- " ...for the Prism Element to where" reads off to me. Should `to` be removed?

Per https://github.com/openshift/openshift-docs/pull/88561#discussion_r1955221375 you might want to check where an ocp cluster to deployed to?

```suggestion
With this release, Nutanix supports more than one subnet for the Prism Element where the {product-title} cluster is deployed. A maximum of 32 subnets for each Prism Element is supported.
```